packages/react/src/combobox/utils.ts

```diff
@@ -90,3 +90,15 @@ export function generateHighlightStyles(
 
 	return styles;
 }
+
+export function useIsIos() {
```

Comment:
Should this be a hook or just a function? Do we care if it runs this on every re-render?

Reply:
I don't think it matters much either way but why not just go for `useMemo` here?

```ts
const isIos = useMemo(
	() =>
		CSS.supports('-webkit-appearance', '-apple-pay-button') &&
		CSS.supports('-webkit-overflow-scrolling', 'auto'),
	[]
);
```

Reply:
That's a great idea